charts/lfx-v2-project-service/templates/deployment.yaml

```
@@ -17,11 +17,19 @@ spec:
         app: {{ .Chart.Name }}
     spec:
       containers:
+        {{- if .Values.image.useLocalImage }}
+        - name: app
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)